lib/module.php

```diff
@@ -275,6 +283,20 @@ function solusiovps_CreateAccount(array $params): string
         $response = $serverResource->create($serverData);
         $payload = Arr::get($response, 'data', []);
 
+        if (empty($params['domain'])) {
+            DB::table('tblhosting')->where('id', $params['serviceid'])->update(['domain' => $payload['name']]);
```

Comment:
let's extract ip to `Hosting` model?
`Hosting::updateByServiceId($params['serviceid'], ['domain' => $payload['name']]);`